File: data/java/pattern_10/code.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner createScanner = new Scanner(System.in);
        int num;
        System.out.print("Enter the number of rows and columns for the square: ");
        num = createScanner.nextInt();

        for (int i = 1; i <= num; i++) {
            for (int j = 1; j <= num - 2; j++) {
                System.out.print(i + " " + j + " ");
            }
            System.out.print("\n");
        }
    }
}

File: data/python/pattern_10/code.py
#MAIN PROGRAM STARTS HERE:
num = int(input('Enter the number of rows and columns for the square: '))

for x in range(1, num + 1):
    for y in range(1, num - 2 + 1):
        print ('{} {} '.format(x, y), end='')
    print()